django/template/base.py

```diff
@@ -151,6 +151,9 @@ def __eq__(self, other):
             self.loader == other.loader
         )
 
+    def __ne__(self, other):
```

Comment:
You're right. We don't have `if six.PY2` for `__ne__` methods anywhere else so let's keep it consistent.